ec2-cluster/modules/grafana/ec2.tf

```diff
@@ -0,0 +1,77 @@
+resource "aws_instance" "grafana" {
+  ami                         = data.aws_ami.amzn2-ami.id
+  instance_type               = var.instance_type
+  associate_public_ip_address = true
+
+  user_data = file("user_data.sh")
```

Comment:
We need to put this file to repo probably or keep single one to share across infrastructure_utils